release-notes/cloudflow-1.3.1.md

```diff
@@ -0,0 +1,42 @@
+# Cloudflow 1.3.1 Release Notes
+
+Cloudflow 1.3.1 is a patch release and contains the following changes:
+
```

Comment:
should we call it a patch release with so many version upgrades ?

Reply:
It's not a major release and it increases the patch level. 
What other term would you suggest?

To me these are just dependency updates instead of major changes. But perhaps I'm underestimating the impact?